plugins/hypervisors/vmware/src/main/java/com/cloud/hypervisor/vmware/manager/VmwareManagerImpl.java

```diff
@@ -459,7 +466,7 @@ private void prepareHost(HostMO hostMo, String privateTrafficLabel) throws Excep
             AboutInfo about = hostMo.getHostAboutInfo();
             if (about != null) {
                 String version = about.getApiVersion();
-                if (version != null && (version.equals("4.0") || version.equals("4.1")) && _portsPerDvPortGroup < DEFAULT_PORTS_PER_DV_PORT_GROUP_VSPHERE4_x) {
+                if (version != null && (version.equals("4.0") || version.equals("4.1")) ) { // && _portsPerDvPortGroup < DEFAULT_PORTS_PER_DV_PORT_GROUP_VSPHERE4_x)
```

Comment:
Why commenting out this condition?

Reply:
I will double check, but IIRC it makes no sense.

Reply:
`_portsPerDvPortGroup < DEFAULT_PORTS_PER_DV_PORT_GROUP_VSPHERE4_x)` comes down to the hardcoded condition `8<256`
```suggestion
                if (version != null && (version.equals("4.0") || version.equals("4.1")) ) { // && _portsPerDvPortGroup < DEFAULT_PORTS_PER_DV_PORT_GROUP_VSPHERE4_x), which means 8 < 256 
```

Reply:
not sure if I should refactor this further, yet.